utils/unminify.js

```diff
@@ -62,11 +65,30 @@ const nilConsumer = {
  * @return {string}
  */
 function normalizeCdnJsUrl(url, version) {
-  const [, origin, rtv, pathname] = cdnJsRegex.exec(url);
-  if (rtv) {
-    return url;
+  const match = cdnJsRegex.exec(url);
+  if (!match) {
+    return;
   }
-  return `${origin}/rtv/${version}${pathname}`;
+
+  const [
+    /* full match */,
+    origin,
+    rtv = version,
+    pathname,
+    ampExtension,
+    module = '',
+    ext,
+  ] = match;
+
+  // We explicitly forbid the experiments and validator "extensions" inside
+  // the v0 directory.
+  if (ampExtension === 'experiments' || ampExtension === 'validator') {
```

Comment:
We're getting error reports from these files?

Reply:
I don't remember why this was added originally.